test/Microsoft.DotNet.Tools.Tests.Utilities/TestBase.cs

```diff
@@ -53,21 +39,16 @@ protected static TestAssets TestAssets
                 {
                     var assetsRoot = Path.Combine(RepoRoot, "TestAssets");
 
-                    s_testAssets = new TestAssets(new DirectoryInfo(assetsRoot)); 
+                    s_testAssets = new TestAssets(
+                        new DirectoryInfo(assetsRoot),
+                        new FileInfo(new EnvironmentProvider().GetCommandPath("dotnet")),
```

Comment:
Can we use the RepoDirectoriesProvider here to, so that it always goes to stage2 without us needing to set the PATH? Or was this intentional?